xLights/ModelFaceDialog.cpp

```diff
@@ -463,6 +474,15 @@ void ModelFaceDialog::SetFaceInfo(Model *cls, std::map< std::string, std::map<st
     }
 
     UpdatePreview("", *wxWHITE);
+    std::list<std::string> warnings = cls->CheckModelSettings();
+    if (!warnings.empty()) {
+        std::string warningsStr;
```

Comment:
join?

Reply:
Thanks .. let me readup and fix this up..

Reply:
There is no join, there is a wxJoin but these are strings. Is there something I am missing?

Reply:
In the utils class Dan wrote a std::strings join method. You could also do it with std::transform and a lamda

Reply:
okay .. I am a rookie here .. it is a std::list but Joins is looking for a std::vector. I would have to scan thru it all again to switch them right? Seems like a waste of cycles. Or is it simpler?
`std::list<std::string> warnings = cls->CheckModelSettings();`

`inline std::string Join(std::vector<std::string> const &strings, std::string delim)`